modules/installation-vsphere-machines.adoc

```diff
@@ -34,7 +49,7 @@ $ base64 -w0 <installation_directory>/worker.ign > <installation_directory>/work
 +
 [source,terminal]
 ----
-$ base64 -w0 <installation_directory>/append-bootstrap.ign > <installation_directory>/append-bootstrap.64
+$ base64 -w0 <installation_directory>/bootstrap.ign > <installation_directory>/bootstrap.64
```

Comment:
I changed the reference to the custom Ignition config. (`s/append-bootstrap.ign/bootstrap.ign`) It was a remnant of the previous steps that we removed in PR27394.

Reply:
Updated to `merge-bootstrap.ign` in this PR because customer was reporting that `append-` was still needed but my understanding is that `append-bootstrap --> merge-bootstrap` starting in 4.6 due to update to the Ignition spec.